lib/samples/add_feature_collection_layer_from_table/add_feature_collection_layer_from_table.dart

```diff
@@ -116,7 +116,7 @@ class _AddFeatureCollectionLayerFromTableState
     // Create a feature collection table for the geometry type with a list of fields and a spatial reference.
     final polylineTable = FeatureCollectionTable(
       fields: [
-        Field.text(name: 'Boundary', alias: 'Boundary Name', length: 50)
+        Field.text(name: 'Boundary', alias: 'Boundary Name', length: 50),
```

Comment:
This comma rule (in multiple places) goes against our code standards so we either need to adopt it/update those, or not have the linter. Will let others weigh in, I don't really mind.

Reply:
I much prefer an automated linter rule, even if it's not exactly what we were doing manually. We can prune it from our coding standard.

Reply:
Here's the linter rule explanation: https://dart.dev/tools/linter-rules/require_trailing_commas
It's going to recommend more commas than what we were doing, but I think that's fine. It has some reasonable exceptions built-in.